Libraries/IntegrationServiceLibrary.psm1

```diff
@@ -741,13 +741,13 @@ Function New-BackupSetup {
 		[String] $HvServer
 	)
 	# Install the Windows Server Backup
-	$winFeatureName = "Windows-Server-Backup"
-	if ((Get-WindowsFeature $winFeatureName).InstallState -ne "Installed") {
-		Write-LogInfo "Installing the $winFeatureName ..."
-		if ( (Add-WindowsFeature $winFeatureName).Success -eq "True" ) {
-			Write-LogInfo "Install the $winFeatureName successfully"
+	$winFeatureName = "WindowsServerBackup"
+	if ((Get-WindowsOptionalFeature -Online -FeatureName $winFeatureName).State -ne "Enabled") {
```

Comment:
Make sure it works with windows 10 server also.

Reply:
Yes, the feature "WindowsServerBackup" should be in the Windows server system. I confirmed in Server 2019 it worked. In Windows 10, there is no "WindowsServerBackup" feature.

Reply:
How about `Get-WindowsFeature to `Get-WindowsOptionalFeature`? Does the `Get-WindowsOptionalFeature` supports windows 10 servers?